pkg/internalregistry/controllers/image_pull_secret_controller.go

```diff
@@ -175,11 +176,17 @@ func (c *imagePullSecretController) sync(ctx context.Context, key string) (error
 	var serviceAccountName = serviceAccountNameForManagedSecret(secret)
 	klog.V(2).InfoS("Refreshing image pull secret", "ns", secret.Namespace, "name", secret.Name, "serviceaccount", serviceAccountName)
 
-	// request new token, bound by default time and bound to this secret
+	// request new token, bound to this secret and with jittered expiration
+	expirationSeconds := 60*60 + rand.Int64N(10*60) // [1h, 1h10m)
```

Comment:
The default ExpirationSeconds is configurable, so we can't just assume it is 1h.

Reply:
> ~~The default ExpirationSeconds is configurable, so we can't just assume it is 1h.~~

I was wrong about that, only the max ExpirationSeconds is configuration.